src/queryparser.py

```diff
@@ -0,0 +1,16 @@
+
+def parse(filter: str, varList: list):
```

Comment:
To be consistent with the module's name I would rename this variable to `query`.

Reply:
Thank you, it was changed